### PR TITLE
Added a feature to view Indian state wise data

### PR DESCRIPTION
**Pull request by inayatm**
Added a new feature to display Indian state covid19 data .
 1)Used external json api
 2)Added new page to display states data: Currently added bootstrap table template with sample data

### CODE REVIEW

File: coronavirus-tracker/src/main/java/com/covid19/services/StateDataService.java [hunk 1]
```diff
@@ -22,10 +29,13 @@
...
 	private static String STATE_DATA_URL="https://api.covid19india.org/state_test_data.json";
+	
+	
+	
 
   @PostConstruct
   @Scheduled(cron="* * 1 * * *")
-  public Object fetchVirusStateData() throws IOException, InterruptedException{
+  public List<StateTestedData> fetchVirusStateData() {
 	  System.out.println("Service calling..");
```
Comment (inayatm2) [inline coronavirus-tracker/src/main/java/com/covid19/services/StateDataService.java]: Remove SOPs

File: coronavirus-tracker/src/main/java/com/covid19/services/StateDataService.java [hunk 2]
```diff
@@ -34,16 +44,13 @@ public Object fetchVirusStateData() throws IOException, InterruptedException{
...
-	   //List<LocationStats> newstats=new ArrayList<>();
-	  for (CSVRecord record : records) {
-	  System.out.println(record.getRecordNumber());
+	  RestTemplate template=new RestTemplate();
+	  
+	  List<StateTestedData> stateTestedData=template.getForObject(STATE_DATA_URL, State.class).getStatedata();
 	  
-	  }
-	 return response.body();
+	  System.out.println(stateTestedData.get(0));
```
Comment (inayatm2) [inline coronavirus-tracker/src/main/java/com/covid19/services/StateDataService.java]: Remove SOPs

File: coronavirus-tracker/target/classes/templates/indianStateData.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+</table>
+<!--Table-->
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+
+  
+  </body>
```
Comment (inayatm2) [inline coronavirus-tracker/target/classes/templates/indianStateData.html]: Add html end tag

**Review by inayatm — COMMENTED** [review]
reviewed and cleaned up the code

File: coronavirus-tracker/target/classes/templates/indianStateData.html (hunk 3 above)
Comment (inayatm) [reply coronavirus-tracker/target/classes/templates/indianStateData.html]: done

File: coronavirus-tracker/src/main/java/com/covid19/services/StateDataService.java (hunk 2 above)
Comment (inayatm) [reply coronavirus-tracker/src/main/java/com/covid19/services/StateDataService.java]: done

**Review by inayatm2 — APPROVED** [review]

**Comment by inayatm2** [comment]
Changes looks good!

**Pull request merged**